misc/linglong/builder/templates/autotools.yaml

```diff
@@ -17,4 +20,12 @@ build:
     build: |
       make ${jobs}
     install: |
-      make ${jobs} DESTDIR=${dest_dir} install
\ No newline at end of file
+      make ${jobs} DESTDIR=${dest_dir} install
+
+enviroment:
+  CFLAGS: |
+    -I${PREFIX}/include -g -O2 -fstack-protector-strong -Wformat -Werror=format-security ${c_flags}
```

Comment:
We should use "-isystem" instead of "-I"

gcc will  scanne `-I options directory` before `standard system directories`（which shuold be ${PREFIX}/include in linglong）. If we use  "-I" for (linglong) standard system directories，the scanning order will be messed up. We should ensure  `standard system directories` be scanned after  "-I" provided by the project itself.

Reply:
Thanks.